### ISSUE TITLE
stylus pen bug

### ISSUE THREAD

**Issue by XPRAMT** [LizardByte/Sunshine#1936]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Using a stylus pen and multitouch may experience intermittent stuttering (approximately every 30 to 60 seconds).
[https://www.youtube.com/watch?v=ZJRAMEsHK3g](https://www.youtube.com/watch?v=ZJRAMEsHK3g)
When checking the log, I discovered that only 7 or 8 stylus packets were received per second during the occurrence of stuttering.

### Host Operating System

Windows

### Operating System Version

22635.2915

### Architecture

64 bit

### Sunshine commit or version

0.21.0.6b7b5996ccd040375b62d930e064abddcc533a72

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

RTX3080

### GPU Driver/Mesa Version

546.33

### Config

```Shell
global_prep_cmd = [{"do":"powershell.exe -executionpolicy bypass -WindowStyle Hidden -file \"C:\\APP\\staging\\ResolutionMatcher.ps1\"","elevated":"false","undo":"powershell.exe -executionpolicy bypass -WindowStyle Hidden -file \"C:\\APP\\staging\\ResolutionMatcher-Functions.ps1\" True"}]
sunshine_name = XPRAMT-Sunshine
key_repeat_delay = 200
upnp = enabled
nvenc_twopass = disabled
fec_percentage = 5
gamepad = x360
nvenc_preset = 2
fps = [60,120,144]
key_repeat_frequency = 60
encoder = nvenc
install_steam_audio_drivers = disabled
resolutions = [
    1280x720,
    1920x1080,
    2560x1080,
    3440x1440,
    1920x1200,
    3840x2160,
    3840x1600,
    2880x1800
]
address_family = both
```

### Relevant log output

```shell
[2023:12:25:22:32:46]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.179167]
y [0.461778]
pressureOrDistance [0.639805]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [244]
tilt [25]
--end pen packet--
[2023:12:25:22:32:46]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.181319]
y [0.458556]
pressureOrDistance [0.642002]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [249]
tilt [24]
--end pen packet--
[2023:12:25:22:32:46]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.184063]
y [0.454889]
pressureOrDistance [0.647375]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [256]
tilt [24]
--end pen packet--
[2023:12:25:22:32:46]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.184514]
y [0.455556]
pressureOrDistance [0.665934]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [258]
tilt [24]
--end pen packet--
[2023:12:25:22:32:46]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.179097]
y [0.465]
pressureOrDistance [0.664957]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [261]
tilt [24]
--end pen packet--
[2023:12:25:22:32:47]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.172986]
y [0.4825]
pressureOrDistance [0.656654]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [258]
tilt [24]
--end pen packet--
[2023:12:25:22:32:47]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.173333]
y [0.496056]
pressureOrDistance [0.65348]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [258]
tilt [24]
--end pen packet--
[2023:12:25:22:32:47]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.174514]
y [0.494444]
pressureOrDistance [0.638584]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [251]
tilt [24]
--end pen packet--
[2023:12:25:22:32:47]: Debug: Frame processing latency (min/max/avg): 6.0ms/7.9ms/6.2ms
[2023:12:25:22:32:47]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.188611]
y [0.474222]
pressureOrDistance [0.647375]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [254]
tilt [24]
--end pen packet--
[2023:12:25:22:32:47]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.198333]
y [0.489833]
pressureOrDistance [0.660562]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [259]
tilt [25]
--end pen packet--
[2023:12:25:22:32:47]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.196667]
y [0.498556]
pressureOrDistance [0.66105]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [254]
tilt [25]
--end pen packet--
[2023:12:25:22:32:47]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.193021]
y [0.494833]
pressureOrDistance [0.659096]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [249]
tilt [27]
--end pen packet--
[2023:12:25:22:32:47]: Debug: Sleep overshoot (min/max/avg): 0.0ms/0.5ms/0.3ms
[2023:12:25:22:32:47]: Debug: NvEnc: encoded frame sizes (min max avg) 0.2 0.8 0.3 kB
[2023:12:25:22:32:47]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.184271]
y [0.4855]
pressureOrDistance [0.650305]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [252]
tilt [25]
--end pen packet--
[2023:12:25:22:32:48]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.185382]
y [0.512111]
pressureOrDistance [0.650305]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [256]
tilt [24]
--end pen packet--
[2023:12:25:22:32:48]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.181354]
y [0.518667]
pressureOrDistance [0.650305]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [24]
--end pen packet--
[2023:12:25:22:32:48]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.189514]
y [0.538222]
pressureOrDistance [0.660806]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [244]
tilt [25]
--end pen packet--
[2023:12:25:22:32:48]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.195486]
y [0.489944]
pressureOrDistance [0.665201]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [245]
tilt [26]
--end pen packet--
[2023:12:25:22:32:48]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.201806]
y [0.525722]
pressureOrDistance [0.673504]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [257]
tilt [25]
--end pen packet--
[2023:12:25:22:32:48]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.196354]
y [0.513944]
pressureOrDistance [0.65812]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [25]
--end pen packet--
[2023:12:25:22:32:48]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.192847]
y [0.528722]
pressureOrDistance [0.652259]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [248]
tilt [26]
--end pen packet--
[2023:12:25:22:32:48]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.202396]
y [0.528667]
pressureOrDistance [0.647619]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [248]
tilt [26]
--end pen packet--
[2023:12:25:22:32:49]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.198229]
y [0.518278]
pressureOrDistance [0.63956]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [244]
tilt [25]
--end pen packet--
[2023:12:25:22:32:49]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.208715]
y [0.543667]
pressureOrDistance [0.666422]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [25]
--end pen packet--
[2023:12:25:22:32:49]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.209861]
y [0.538]
pressureOrDistance [0.652015]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [244]
tilt [25]
--end pen packet--
[2023:12:25:22:32:49]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.213889]
y [0.555333]
pressureOrDistance [0.654457]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [242]
tilt [25]
--end pen packet--
[2023:12:25:22:32:49]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.217708]
y [0.538389]
pressureOrDistance [0.655678]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [243]
tilt [26]
--end pen packet--
[2023:12:25:22:32:49]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.20684]
y [0.543778]
pressureOrDistance [0.654457]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [244]
tilt [25]
--end pen packet--
[2023:12:25:22:32:49]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.216563]
y [0.528222]
pressureOrDistance [0.660317]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [24]
--end pen packet--
[2023:12:25:22:32:50]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.20559]
y [0.525056]
pressureOrDistance [0.633211]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [245]
tilt [26]
--end pen packet--
[2023:12:25:22:32:50]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.213889]
y [0.544611]
pressureOrDistance [0.652259]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [25]
--end pen packet--
[2023:12:25:22:32:50]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.221528]
y [0.531]
pressureOrDistance [0.639316]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [245]
tilt [26]
--end pen packet--
[2023:12:25:22:32:50]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.217257]
y [0.540889]
pressureOrDistance [0.643223]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [244]
tilt [25]
--end pen packet--
[2023:12:25:22:32:50]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.227674]
y [0.5385]
pressureOrDistance [0.662515]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [25]
--end pen packet--
[2023:12:25:22:32:50]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.221944]
y [0.518]
pressureOrDistance [0.643223]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [244]
tilt [25]
--end pen packet--
[2023:12:25:22:32:50]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.228958]
y [0.545722]
pressureOrDistance [0.66569]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [25]
--end pen packet--
[2023:12:25:22:32:50]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.239965]
y [0.540722]
pressureOrDistance [0.658364]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [245]
tilt [26]
--end pen packet--
[2023:12:25:22:32:51]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.2425]
y [0.521833]
pressureOrDistance [0.654212]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [244]
tilt [24]
--end pen packet--
[2023:12:25:22:32:51]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.254271]
y [0.528056]
pressureOrDistance [0.665201]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [249]
tilt [24]
--end pen packet--
[2023:12:25:22:32:51]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.264514]
y [0.518389]
pressureOrDistance [0.677167]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [250]
tilt [23]
--end pen packet--
[2023:12:25:22:32:51]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.266181]
y [0.508722]
pressureOrDistance [0.664957]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [249]
tilt [25]
--end pen packet--
[2023:12:25:22:32:51]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.271736]
y [0.536111]
pressureOrDistance [0.694017]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [253]
tilt [23]
--end pen packet--
[2023:12:25:22:32:51]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.279687]
y [0.522833]
pressureOrDistance [0.68718]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [248]
tilt [23]
--end pen packet--
[2023:12:25:22:32:51]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.285313]
y [0.507]
pressureOrDistance [0.68083]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [249]
tilt [24]
--end pen packet--
[2023:12:25:22:32:52]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.296458]
y [0.531389]
pressureOrDistance [0.683272]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [250]
tilt [23]
--end pen packet--
[2023:12:25:22:32:52]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.304514]
y [0.531444]
pressureOrDistance [0.682784]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [24]
--end pen packet--
[2023:12:25:22:32:52]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.308472]
y [0.510444]
pressureOrDistance [0.683517]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [246]
tilt [24]
--end pen packet--
[2023:12:25:22:32:52]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.310104]
y [0.507722]
pressureOrDistance [0.679121]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [249]
tilt [24]
--end pen packet--
[2023:12:25:22:32:52]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.312569]
y [0.530778]
pressureOrDistance [0.69475]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [252]
tilt [25]
--end pen packet--
[2023:12:25:22:32:52]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.319479]
y [0.517722]
pressureOrDistance [0.693285]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [250]
tilt [26]
--end pen packet--
[2023:12:25:22:32:52]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.32309]
y [0.493278]
pressureOrDistance [0.686447]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [246]
tilt [24]
--end pen packet--
[2023:12:25:22:32:52]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.328021]
y [0.529333]
pressureOrDistance [0.698901]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [253]
tilt [23]
--end pen packet--
[2023:12:25:22:32:53]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.333056]
y [0.525778]
pressureOrDistance [0.703053]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [249]
tilt [25]
--end pen packet--
[2023:12:25:22:32:53]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.336944]
y [0.491333]
pressureOrDistance [0.691575]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [24]
--end pen packet--
[2023:12:25:22:32:53]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.340764]
y [0.511667]
pressureOrDistance [0.695482]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [253]
tilt [23]
--end pen packet--
[2023:12:25:22:32:53]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.343889]
y [0.5335]
pressureOrDistance [0.708669]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [253]
tilt [23]
--end pen packet--
[2023:12:25:22:32:53]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.350243]
y [0.499278]
pressureOrDistance [0.701099]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [247]
tilt [25]
--end pen packet--
[2023:12:25:22:32:53]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.355347]
y [0.503]
pressureOrDistance [0.696703]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [249]
tilt [24]
--end pen packet--
[2023:12:25:22:32:53]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.360556]
y [0.533722]
pressureOrDistance [0.707692]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [252]
tilt [23]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.367743]
y [0.524444]
pressureOrDistance [0.710867]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [251]
tilt [24]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.371875]
y [0.486333]
pressureOrDistance [0.699634]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [248]
tilt [23]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.377569]
y [0.533833]
pressureOrDistance [0.707692]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [255]
tilt [22]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.370972]
y [0.486778]
pressureOrDistance [0.690843]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [246]
tilt [24]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.397813]
y [0.528167]
pressureOrDistance [0.704762]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [258]
tilt [23]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.388576]
y [0.489722]
pressureOrDistance [0.693529]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [255]
tilt [22]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.410035]
y [0.506333]
pressureOrDistance [0.704762]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [255]
tilt [22]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.416701]
y [0.529556]
pressureOrDistance [0.711844]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [257]
tilt [22]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.411215]
y [0.544556]
pressureOrDistance [0.715018]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [257]
tilt [22]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.397257]
y [0.527389]
pressureOrDistance [0.689377]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [258]
tilt [23]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.399514]
y [0.503222]
pressureOrDistance [0.68547]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [255]
tilt [22]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.406354]
y [0.494778]
pressureOrDistance [0.683028]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [255]
tilt [22]
--end pen packet--
[2023:12:25:22:32:54]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.411875]
y [0.495056]
pressureOrDistance [0.684005]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [257]
tilt [22]
--end pen packet--
[2023:12:25:22:32:55]: Debug: --begin pen packet--
eventType [03]
toolType [01]
penButtons [00]
x [0.419549]
y [0.505944]
pressureOrDistance [0.694994]
contactAreaMajor [0]
contactAreaMinor [0]
rotation [257]
tilt [22]
--end pen packet--
```

**Comment by XPRAMT** [comment]
Stuttering still exists

**Comment by rasmus-z** [comment]
> Stuttering still exists

Which client are you using?

**Comment by ClassicOldSong** [comment]
If you're using the Android client, it's potentially fixed here: https://github.com/ClassicOldSong/moonlight-android/releases/tag/v12.1-240714-NIOR

You need to uncheck `Enable battery status report for gamepad` in the preference.

**Comment by ReenigneArcher** [comment]
@XPRAMT is this still an issue in the latest pre-releases?

**Comment by cfalas** [comment]
I also used to experience this bug, but haven't noticed any stuttering since the change to inputtino, but waiting for @XPRAMT to confirm as well

**Comment by XPRAMT** [comment]
I updated Sunshine to version 2024.730.191523, and it seems the bug has been fixed.

**Comment by ReenigneArcher** [comment]
Fixed by #2606

**Comment by Douvahkiin** [comment]
The problem still exists now.
